doc/man/cockpit.conf.xml

```diff
@@ -76,6 +76,20 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
 <programlisting language="js">
 [WebService]
 ProtocolHeader = X-Forwarded-Proto
+</programlisting>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+        <varlistentry>
+          <term><option>ForwardedForHeader</option></term>
+          <listitem>
+            <para>Configure cockpit to look at the contents of this header to determine the real origin of a
```

Comment:
Is `X-F-F` some reasonably widely accepted standard? Does nginx/Apache etc. have some recommendations or builtin defaults around that? Then the documentation should recommend to use X-F-F.

Reply:
So the story here is that there's a standardised "Forwarded" header that we all ought to be using since 8 years ago, but nobody supports it and everyone is still using X-Forwarded-For in the wild.  I looked for examples of how to get nginx to add the Forwarded header for example, and I got some really complicated suggestions, whereas nginx has already-built-in support for X-Forwarded-For.  For other servers that I checked I did searches like "squid reverse proxy forwarded header" and landed on pages that were just talking about X-Forwarded-For, seemingly oblivious to the existence of the standard.

tl;dr: "Forwarded" is the official standard, but nobody cares.  X-Forwarded-For is the (absolutely pervasive) de facto standard for this.  If there were to be a second place finisher, it would be X-Real-IP.

Reply:
Ack, thanks. So let's add something like

> Most web servers use the `X-Forwarded-For` header name for this purpose.

to this documentation?